doc/nrf/ug_app_dev.rst

```diff
@@ -0,0 +1,72 @@
+.. _ncs-app-dev:
+
+Application Development
+#######################
+
+This page describes the fundamentalas about developing a new application based
+on the |NCS|.
+
+Build and Configuration System
+******************************
+
+Zephyr's build and configuration system
+=======================================
+
+The |NCS| build and configuration system is in fact Zephyr's, which in
```

Comment:
I would say "based on Zephyr's" since multi-image and partition manager are pretty significant out of tree additions.